src/doRequest.js

```diff
@@ -1,16 +1,17 @@
 export default function doRequest(method, path, body = null, headers = {}) {
```

Comment:
This would be better as object destructuring, so you don't have to remember the order of the params. Four is quite a few.

Reply:
Done in 06e798b.